fixtures/response/user_response_suggestions.py

```diff
@@ -30,7 +30,7 @@
 
 make_suggestion_in_non_existent_room = '''
 mutation{
-  createResponse(questionId:3, roomId:90, textArea:"Any other suggestion") {
+  createResponse(responses: [{questionId:3, textArea:"Any other suggestion"}], roomId:90) { # noqa
```

Comment:
You can easily remove this `# noqa` by having your code within 
```
createResponse(responses: [
{
the parameters in here
}
]
```

Reply:
Thanks, it has been done.